pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -104,6 +104,10 @@ self: super: {
   # Fix test trying to access /home directory
   shell-conduit = (overrideCabal super.shell-conduit (drv: {
     postPatch = "sed -i s/home/tmp/ test/Spec.hs";
+
+    # the tests for shell-conduit on Darwin illegitimatey assume non-GNU echo
+    # see: https://github.com/psibi/shell-conduit/issues/12
+    doCheck = !pkgs.stdenv.hostPlatform.isDarwin;
```

Comment:
It might be possible to remedy this issue by adding `pkgs.coreutils` into `testToolDepends` for this package so that GNU echo is in $PATH. It's not essential ... just an idea.

Reply:
The problem is the reverse: because GNU echo is on the path, the tests don't find the behavior they expect on Darwin with Nix.

Reply:
Ah, I see. Sorry, I misread the comment above.